definitions/procedures/sync_plans.rb

```diff
@@ -0,0 +1,34 @@
+class Procedures::SyncPlans < ForemanMaintain::Procedure
+  metadata do
+    for_feature :sync_plans
+    description 'enable/disable sync plans before/after upgrade'
+  end
+
+  attr_reader :rollback
+
+  def initialize(options = {})
+    @rollback = options[:rollback]
+  end
+
+  def run
+    if rollback
+      enabled_sync_plans
+    else
+      disable_all_enabled_sync_plans
+    end
+  end
+
+  private
+
+  def disable_all_enabled_sync_plans
+    with_spinner('disabling all sync plans') do |spinner|
```

Comment:
Since even for the enable case, there is one request per sync plan, having that behind a spinner might be a good idea